packages/fxa-settings/src/lib/metrics.ts

```diff
@@ -161,10 +180,14 @@ export function init(flowQueryParams: FlowQueryParams) {
         redirectPath += window.location.search;
       }
 
+      const search = window.location.search
+        ? `${window.location.search}&redirect_to=${encodeURIComponent(
+            redirectPath
+          )}`
+        : `?redirect_to=${encodeURIComponent(redirectPath)}`;
```

Comment:
The redirect need to pass the query params to content server since content server's app-start rely on some of those values.  This change allows us to get the correct `service` value at the very least.